src/components/existing-request/existing-request-search.vue

```diff
@@ -149,36 +149,47 @@ export default class ExistingRequestSearch extends Vue {
     }
   }
 
-  private emailRules = [
-    v => (!!v || !!this.getExistingRequestSearch?.phoneNumber) || 'Please enter either the phone or the email',
-    v => !!this.getExistingRequestSearch?.phoneNumber || (!!v && /.+@.+\..+/.test(v)) ||
-      'Please be sure to enter a valid email'
-  ]
   private nrRules = [
     v => this.NR_REGEX.test(v) || 'Please enter a valid NR number'
   ]
-  private phoneRules = [
-    v => (!v || v.length <= 30) || 'Cannot exceed 30 characters'
-  ]
+  private emailRules = []
+  private phoneRules = []
 
   private async handleSubmit (): Promise<void> {
+    this.applyRules()
     this.$refs.existingNrForm.validate()
     await this.$nextTick()
+
     if (this.isValid) {
       await this.findNameRequest(null)
       if (this.getNr?.failed) {
         // capture error text and then clear out the NR data
         this.errorMessage = this.getNr.text
         this.setNameRequest({})
       }
-      // FUTURE: clear out applicant's phone and email ?
-      // for (let key of ['emailAddress', 'phoneNumber']) {
-      //   let value = ''
-      //   newReqModule.mutateExistingRequestSearch({ key, value })
-      // }
     }
   }
 
+  /** Apply the field rules to request search inputs. */
+  private applyRules (): void {
+    const { emailAddress, phoneNumber } = this.getExistingRequestSearch
+
+    this.phoneRules = (phoneNumber || !emailAddress)
+      ? [
+        v => !!v || 'Please enter a phone number',
```

Comment:
Isn't this rule redundant, since we are in the if block where `phoneNumber` isn't falsy?

Reply:
Oddly, this seems to work -- I can get both validation errors if I enter neither and click the button, which I think is correct.
I think it's odd that this works because this logic is working with 2 sets of variables: `v` in the validation functions which is injected by each component itself, and phoneNumber/emailAddress which comes from the store. I'm thinking there's still an async issue here (though it's hidden and so maybe we don't care).
Thoughts?

Reply:
Partly true. In the event the user hasn't entered either an email or phone this will trigger for both fields.

Reply:
Yup, yup. OK then.